Lib/sqlite3/test/userfunctions.py

```diff
@@ -283,6 +283,8 @@ def CheckFuncNonDeterministic(self):
         self.assertEqual(mock.call_count, 2)
 
     @unittest.skipIf(sqlite.sqlite_version_info < (3, 8, 3), "deterministic parameter not supported")
+    @unittest.skipIf(True, 'Broken skip until upstream fix is applied'
```

Comment:
We can drop this commit, since we'll rebase after this is merged

Reply:
OK